web/ui/mantine-ui/src/pages/query/DataTable.module.css

```diff
@@ -1,10 +1,23 @@
 .tableWrapper {
-  border: 1px solid
-    light-dark(var(--mantine-color-gray-3), var(--mantine-color-dark-5));
-  border-radius: var(--mantine-radius-default);
+    border: 1px solid
```

Comment:
Hmm, might have something to do with eslint rules?
I suggest we copy the ones in the current UI, they work really well and I've never had issues

Reply:
> Hmm, might have something to do with eslint rules?

Honestly, I'm not sure - we're doing 2 spaces in both the old and new code bases, and my editor is correctly doing 2 spaces in both cases without a specific rule for it (as far as I'm aware). But that may just be because it's the VSCode default, and adding an explicit rule for it would be even better.

> I suggest we copy the ones in the current UI, they work really well and I've never had issues

If you're talking about https://github.com/prometheus/prometheus/blob/mantine-ui/web/ui/react-app/.eslintrc.json vs. https://github.com/prometheus/prometheus/blob/mantine-ui/web/ui/mantine-ui/.eslintrc.cjs, I'd be happy to copy over specific rules that we like, but a bit wary copying over old cruft that we don't know if we still want.